README.rst

```diff
@@ -1,39 +1,41 @@
-aldryn-style
+============
+Aldryn Style
 ============
 
-A Plugin for django CMS to add CSS classes to other plugins
-
+A Plugin for django CMS to add CSS classes to other plugins.
 
+------------
 Installation
 ------------
 
 This plugin requires `django CMS` 2.4 or higher to be properly installed.
 
-* In your projects `virtualenv`_, run ``pip install aldryn-style``.
+* Create a `virtualenv` and run ``pip install aldryn-style``
 * Add ``'aldryn_style'`` to your ``INSTALLED_APPS`` setting.
 * Run ``manage.py migrate aldryn_style``.
 
-
+-----
 Usage
 -----
 
 You can define styles in your settings.py:
 
 ```
+gettext = lambda s: s
 ALDRYN_STYLE_CLASS_NAMES = (
-    ('info', _("info")),
-    ('new', _("new")),
-    ('hint', _("hint")),
+    ('info', gettext('info')),
+    ('new', gettext('new')),
+    ('hint', gettext('hint')),
 )
```

Comment:
the generated setting from django 1.7 adds this: ``gettext = lambda s: s`` check: https://github.com/divio/django-cms-demo/blob/master/src/settings.py#L13